Makefile

```diff
@@ -102,7 +102,7 @@ generate:
 .PHONY: generate-swagger
 generate-swagger:
 	@hash swagger > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
-		$(GO) get -u github.com/go-swagger/go-swagger/cmd/swagger; \
+		GO111MODULE="on" $(GO) get -u github.com/go-swagger/go-swagger/cmd/swagger@latest; \
```

Comment:
May we should also change the `@latest` to `@v0.19.0`. Currently they are the same, but if a new version is released CI may breaks again.

Reply:
changed